src/App.js

```diff
@@ -19,13 +20,32 @@ export default function App() {
                 item.name.toLowerCase().includes(userInput.toLowerCase())
               )
               .map(({ name, image }) => (
-                <AttractionCard key={name} name={name} image={image} />
+                <AttractionCard
+                  key={name}
+                  name={name}
+                  image={image}
+                  likePlace={likePlace}
+                  likedPlaces={likedPlaces}
+                />
               ))}
           </CardLayout>
         ))}
       </AppLayout>
     </>
   )
+
+  function likePlace(currentId) {
+    let newArray
+
+    if (likedPlaces.includes(currentId)) {
+      newArray = likedPlaces.filter(id => id !== currentId)
+    } else {
+      newArray = [...likedPlaces, currentId]
+    }
+
+    setLikedPlaces(newArray)
+    console.log(newArray)
```

Comment:
We always want to main branch to be nice and clean. Comments and `console.log` should be excluded.

Reply:
Thank you, forgot to delete it after testing my function